source/manage-files.rst

```diff
@@ -0,0 +1,191 @@
+
+   From the file's Details view:
+
+   - You can :ref:`Download <download>` the file.
+
+   - You can :ref:`Delete <delete>` the file.
+
+   - You can view its Details.
+
+   - You can modify the Permissions/Access to the File.
+
+.. _delete:
+
+Delete Folders and Files
+========================
+
+You can delete a folder or a file from its :ref:`Details view
+<view-details>`:
+
+#. Open the Details View for the folder or file to delete:
+
+   - For a folder, navigate to the folder and click :guilabel:`Folder
+     Details`. This opens its Details View.
+
+   - For a file, select the file. This opens its Details View.
+
+#. From the Details view, click :guilabel:`Delete`. A Confirmation
+   dialog appears. 
+
+#. To confirm the delete, click :guilabel:`OK` in the Confirmation
+   dialog.
+
+.. _download:
+
+Download Files
+==============
+
+You can download a file from its :ref:`Details view <view-details>`:
+
+#. Select the file to download. This opens its Details View.
+
+#. From the Details view, click :guilabel:`Download`. Follow your Web
+   browser's download action flow.
+
+.. _modify-permissions:
+
+Modify Permissions for Folders and Files
+========================================
+
+You can modify the access permissions for a folder or a file from its
+:ref:`Details view <view-details>`:
+
+#. Open the Details View:
+
+   - For a folder, navigate to the folder and click
+     :guilabel:`Folder Details`. This opens its Details View.
+
+   - For a file, select the file. This opens its Details View.
+
+#. From the Details view, click :guilabel:`Permissions` to expand the
+   panel.
+
+   - To grant access to a new individual, 
+
+     a. Click :guilabel:`Add with WebID`.
+
+     #. Enter the individual's WebID. You must include specify the full
+        URL, including the protocol ``https:// ...``.
+
+     #. Expand the access modes by clicking on the Access Modes caret (``v``).
+
+     #. Select the access modes to grant.
+
+     #. Click :guilabel:`Add`.
+
+   - To modify existing access for an individual,
+   
+     a. Expand the individual's access modes by clicking on the associated
+        Access Modes caret (``v``).
+     
+        - To revoke an access mode, deselect the access mode to revoke.
+
+        - To grant an access mode, select the access mode to grant.
+
+     #. Click :guilabel:`SAVE`.
+
+Create Folder
+=============
+
+To create a folder in the current folder:
+
+#. Click on the :guilabel:`Create File` link.
+
+#. Enter in the folder's name.
+
+#. Click :guilabel:`Create Folder`.
+
+Upload Files
+============
+
+To upload a file to the current folder:
+
+#. Click on the :guilabel:`Upload File` link.
+
+#. Select the file to upload.
+
+.. _files-view-bookmark:
+
+Bookmark Files and Folders
+==========================
+
+From the Files view, you can bookmark a folder or a file by selecting
+the star icon next to the folder or file. To remove the folder or file
+from your bookmarks, deselect the star icon.
+
```

Comment:
If you want bookmark the folder you are currently viewing:

#. Click the Pod Indicator dropdown just below the Account menu.

#.  Select the :guilabel:`Bookmark Pod` option